itou/siaes/tests/tests_import_siae_command.py

```diff
@@ -123,51 +123,83 @@ def test_creatable_conventions_inactive_siae(self):
         )
         self.assertEqual((siae.source, siae.siret, siae.kind), (Siae.SOURCE_ASP, SIRET, Siae.KIND_ACI))
 
-    def test_check_signup_possible(self):
-        def set_inactive(membership):
-            """Set the membership inactive in the "has_members" meaning, see organization
-            abstract models for details
-            """
-            user = membership.user
-            user.is_active = False
-            user.save(update_fields=["is_active"])
-            membership.is_active = False
-            membership.save(update_fields=["is_active"])
-
-        # has to be lazy-loaded to benefit from the file mock, this management command does crazy stuff at import
+    def set_inactive(self, membership):
+        """
+        Helper used to set the membership inactive in the "has_members" meaning,
+        see organization abstract models for details.
+        """
+        user = membership.user
+        user.is_active = False
+        user.save(update_fields=["is_active"])
+        membership.is_active = False
+        membership.save(update_fields=["is_active"])
+
+    def get_import_siae_command_instance(self):
+        # Has to be lazy-loaded to benefit from the file mock, this management command does crazy stuff at import.
         from itou.siaes.management.commands import import_siae
 
         instance = import_siae.Command()
+        # Required otherwise the variable is undefined and throws an exception when incrementend the first time.
         instance.fatal_errors = 0
+        return instance
 
-        # an SIAE with no members but an auth email: no errors
+    def test_check_signup_possible_for_a_siae_without_members_but_with_auth_email(self):
+        """
+        There is an auth_email thus regular signup is possible, no error.
+        """
+        instance = self.get_import_siae_command_instance()
         SiaeFactory(auth_email="tadaaa")
         with self.assertNumQueries(1):
             instance.check_whether_signup_is_possible_for_all_siaes()
         self.assertEqual(instance.fatal_errors, 0)
 
-        # an SIAE with no members and no auth email: 1 error
+    def test_check_signup_possible_for_a_siae_without_members_nor_auth_email(self):
+        """
+        There is no auth_email thus no regular signup,
+        and no member thus no invitation possible.
+        This should throw an error.
+        """
```

Comment:
pareil. Ce commentaire réexplique soit le titre soit le contenu du test, donc il va au mieux être juste et se répéter, au pire mal vieillir et mettre le doute au relecteur.

https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/

Je serai difficile à convaincre du contraire là dessus et j'ai deja cité cet article (il y en a bien d'autres, lire Clean Code de Martin par exemple) N fois, mais je continuerai !